harness/determined/common/api/_session.py

```diff
@@ -1,16 +1,18 @@
 import abc
+import copy
 import json as _json
-from typing import Any, Dict, Optional, Tuple, Union
+from typing import Any, Dict, Optional, Tuple, TypeVar, Union
 
 import requests
 import urllib3
 
```

Comment:
I think this will also need to be updated 

 https://github.com/determined-ai/determined/blob/bc1b431734f396066330508fe6250546c6304dfe/.circleci/scripts/wait_for_perf_migration_upload_results.py#L38

Reply:
Fixed. I hadn't known that `**/*.py` glob skipped hidden directories. Thanks for catching this.